_deploy/r/gnoswap/gns/gns_test.gno

```diff
@@ -0,0 +1,249 @@
+package gns
+
+import (
+	"fmt"
```

Comment:
```suggestion
	"gno.land/p/demo/ufmt"
```

make sure to remove all of the parts used below.

Reply:
we can use `fmt` for test case, do you still prefer to change it

Reply:
Well, it's not a big problem. just leave it alone to avoid more work